debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/LegacyV1MySqlSourceInfoStructMaker.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+
+import io.debezium.config.CommonConnectorConfig;
+import io.debezium.connector.LegacyV1AbstractSourceInfoStructMaker;
+
+public class LegacyV1MySqlSourceInfoStructMaker extends LegacyV1AbstractSourceInfoStructMaker<SourceInfo> {
+
+    private final Schema schema;
+
+    public LegacyV1MySqlSourceInfoStructMaker(String connector, String version, CommonConnectorConfig connectorConfig) {
+        super(connector, version, connectorConfig);
+        schema = commonSchemaBuilder()
+                .name("io.debezium.connector.mysql.Source")
+                .field(SourceInfo.SERVER_ID_KEY, Schema.INT64_SCHEMA)
```

Comment:
@jpechane, it seems this is missing `.field(SERVER_NAME_KEY, Schema.STRING_SCHEMA)` which was there before? Can you double check?

Reply:
@gunnarmorling Thanks for spotting, this was lost during refactoring and unfrtunately MySQL lacked the schema test. This is added now.

Reply:
Cool; thanks!